go_to_office/lib/admin/officesList.dart

```diff
@@ -0,0 +1,85 @@
+import 'package:flutter/material.dart';
+import 'office.page.dart';
+import '../strings.dart';
+
+void main() {
+  runApp(new MaterialApp(
+    title: "Offices",
+    home: new OfficesListPage(),
+  ));
+}
+final List<Office> officesList = [
+  Office(name: 'Herzeliya', description: 'Herzeliya Office', id: '1'),
+  Office(name: 'Budapest', description: 'Budapest Office', id: '2'),
+  Office(name: 'Boston', description: 'Boston Office', id: '3')];
+
+class OfficesListPage extends StatefulWidget {
+  @override
+  State<StatefulWidget> createState() => _OfficesListPageState();
+}
+
+class _OfficesListPageState extends State<OfficesListPage> {
+  @override
+  Widget build(BuildContext context) {
+
+    // Object officateAssreddress = {};
+    return Scaffold(
+        appBar: AppBar(
+          title: Text("Offices"),
+        ),
+        body: SingleChildScrollView(
+          padding: EdgeInsets.all(15.0),
+          child: Column (
+            mainAxisSize: MainAxisSize.min,
+             children: <Widget>[
+              ElevatedButton(
+                child: Text('Add New Office'),
+                onPressed: () {
+                  Navigator.push(
+                    context,
+                    MaterialPageRoute(builder: (context) => OfficePage(null)),
+                  );
+                },
+              ),
+              ListView.builder(
+                scrollDirection: Axis.vertical,
+                shrinkWrap: true,
+                padding: const EdgeInsets.all(8),
+                itemCount: officesList.length,
+                itemBuilder: (BuildContext context, int index) {
+                  return new GestureDetector(
+                    onTap: () => openOfficePage(officesList[index].id),
```

Comment:
null check? can it be problematic?

Reply:
If the officesList is empty the loop will not run